packages/corvid-e2e-tests/test/drivers/localEditorDriver.js

```diff
@@ -35,17 +35,13 @@ module.exports = page => {
   };
 
   const login = async ({ username, password }) => {
-    const emaiInput = await page.$("#input_0");
+    const emaiInput = await page.$('input[name="email"]');
```

Comment:
I wonder, do we know if perhaps the team in-charge of the login page has such drivers? Would be great if they did, to make sure that we are always in sync with any possible changes they make there

Reply:
could be nice... i'll ask them